package.json

```diff
@@ -8,7 +8,7 @@
     "clean": "rm -rf dist",
     "dev": "NODE_ENV=development node server.js",
     "lint": "npm run lint:check -- --fix",
-    "lint:check": "eslint --ext .js,.vue client server",
+    "lint:check": "eslint --ext .js,.vue client server LICENSE",
```

Comment:
adds linter for LICENSE file at root. This will allow auto-generation for LICENSE. Will also fail lint if changed and not passing linting rules.